src/Tasks/Microsoft.Common.CrossTargeting.targets

```diff
@@ -22,6 +23,9 @@ Copyright (C) Microsoft Corporation. All rights reserved.
   <Target Name="_ComputeTargetFrameworkItems" Returns="@(InnerOutput)">
     <ItemGroup>
       <_TargetFramework Include="$(TargetFrameworks)" />
```

Comment:
Renaming `_TargetFramework` to `_TargetFrameworks` would make it clear there are multiple tfms